macros/edr/dbt_artifacts/upload_run_results.sql

```diff
@@ -1,12 +1,11 @@
 {% macro upload_run_results(results) %}
+    {% if elementary.get_config_var('disable_run_results') %}
+        {% do elementary.edr_log("Run results are disabled, skipping upload.") %}
```

Comment:
I think we want to do it because once the on-run-end hook is not added by the user, we want to communicate to him that it exists and runs and works. Also if it fails, we want them to understand that the failure was part of that process. 
@oravi @elongl